### PR TITLE
Bug- Track View in Sequence Editor

### PR DESCRIPTION
**Pull request by tjmgd**
Previously, in the Track View of the Sequence Editor, if a user selected a track, and then switched to No-Sequence, the previous track's data would be displayed. This is misleading.

This fix resolves that issue by removing the data from the Track View window, as appropriate.

### CODE REVIEW

File: Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/Commands.cpp [hunk 1]
```diff
@@ -588,6 +588,8 @@ namespace EMStudio
             {
                 return false;
             }
+
+            animGraph->SetDirtyFlag(false);
```
Comment (amzn-rhhong) [inline Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/Commands.cpp:592]: this fix is in conflict with this one https://github.com/o3de/o3de/pull/5936
5936 also check the update flag which is the proper way. I suggest take this change out and wait for 5936 submitted.

File: Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAssetCtrl.cpp [hunk 2]
```diff
@@ -1072,10 +1072,10 @@ namespace AzToolsFramework
             RefreshAutocompleter();
         }
 
-        // When focus is lost, clear the field if necessary
+        // When focus is lost, revert to the selected asset
         if (!focus && m_incompleteFilename)
         {
-            HandleFieldClear();
+            SetSelectedAssetID(GetCurrentAssetID());
         }
```
Comment (AMZN-daimini) [inline Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAssetCtrl.cpp:1079]: This PR seems to be including an unrelated commit, can you try to merge latest and see if that fixes it?

**Comment by tjmgd** [comment]
Closing PR based on comments
